signup.css

```
@@ -0,0 +1,122 @@
+* {
+    box-sizing: border-box;
+}
+
+/* 버튼 관련 */
+.sign-up-button {
+    width: 100%;
+    height: 56px;
+    border-radius: 40px;
+    background-color: #3692FF;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    color: #F3F4F6;
+    margin-bottom: 24px;
+    font-size: 20px;
+    font-weight: 600;
+}
+
+/* 전체적인 틀 */
+body {
+    font-family: "Pretendard Variable", Pretendard, sans-serif;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+container {
```

❗️ 수정요청
container라는 태그는 없으니 오타이신 것 같아요~

```suggestion
.container {
```

---

이부분은 body 다음에 가장 크게 container 로 한 번 감싸주었습니다. 아마도 나중에 큰 틀을 정리하기 쉽게 하기 위한 것 같습니다. 이렇게 큰 태그로 한 번 감싸주는것이 괜찮을까요?